docs/js-core-1/week-1/lesson.md

```diff
@@ -32,6 +33,7 @@ By the end of this class, you should be able to:
 - Call functions within functions
 - Search and read documentation to help when you are stuck
 - Use the Code Style Guide to give feedback on other trainees work
+- Understand the core concepts of the agile software development methodology
```

Comment:
```suggestion
- Describe four core concepts of the Agile software development methodology
```

Understand is an overarching term. In a learning objective, choose an active verb that belongs to the understanding group. Bloom's taxonomy may help you here: https://bloomstaxonomy.net/

Reply:
Deleting PD learning objectives and not adding any new ones until they are re-created by the PD syllabus review.